app/models/reservation.rb

```diff
@@ -9,6 +9,14 @@ class Reservation < ActiveRecord::Base
   include Reservations::RelaySupport
   include Reservations::MovingUp
 
+  CATEGORIES = %w(
```

Comment:
@drench can probably speak to this better, but it feels like this definition should be on `AdminReservation`.